service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/StønadsstatistikkMapper.kt

```diff
@@ -0,0 +1,93 @@
+package no.nav.su.se.bakover.service.statistikk
+
+import no.nav.su.se.bakover.common.Tidspunkt
+import no.nav.su.se.bakover.common.log
+import no.nav.su.se.bakover.common.zoneIdOslo
+import no.nav.su.se.bakover.domain.AktørId
+import no.nav.su.se.bakover.domain.behandling.Behandling
+import no.nav.su.se.bakover.domain.beregning.Månedsberegning
+import no.nav.su.se.bakover.domain.grunnlag.Grunnlag
+import no.nav.su.se.bakover.domain.revurdering.IverksattRevurdering
+import no.nav.su.se.bakover.domain.vedtak.Vedtak
+import no.nav.su.se.bakover.domain.vedtak.VedtakType
+import java.time.Clock
+import java.time.LocalDate
+
+internal class StønadsstatistikkMapper(clock: Clock) {
+    val nå = Tidspunkt.now(clock)
+
+    fun map(vedtak: Vedtak.EndringIYtelse, aktørId: AktørId, ytelseVirkningstidspunkt: LocalDate): Statistikk.Stønad {
+        return Statistikk.Stønad(
+            funksjonellTid = nå,
+            tekniskTid = nå,
+            stonadstype = Statistikk.Stønad.Stønadstype.SU_UFØR,
+            sakId = vedtak.behandling.sakId,
+            aktorId = aktørId.toString().toLong(),
+            sakstype = vedtakstype(vedtak.vedtakType),
+            vedtaksdato = vedtak.opprettet.toLocalDate(zoneIdOslo),
+            vedtakstype = vedtakstype(vedtak.vedtakType),
+            vedtaksresultat = when (vedtak.vedtakType) {
+                VedtakType.SØKNAD -> Statistikk.Stønad.Vedtaksresultat.INNVILGET
+                VedtakType.ENDRING -> Statistikk.Stønad.Vedtaksresultat.INNVILGET
+                VedtakType.OPPHØR -> Statistikk.Stønad.Vedtaksresultat.OPPHØRT
+                else -> throw RuntimeException("Ugyldig vedtaksresultat")
+            },
+            behandlendeEnhetKode = "4815",
+            ytelseVirkningstidspunkt = ytelseVirkningstidspunkt,
+            gjeldendeStonadVirkningstidspunkt = vedtak.behandling.periode.fraOgMed,
+            gjeldendeStonadStopptidspunkt = vedtak.behandling.periode.tilOgMed,
+            gjeldendeStonadUtbetalingsstart = vedtak.behandling.periode.fraOgMed,
+            gjeldendeStonadUtbetalingsstopp = vedtak.behandling.periode.tilOgMed,
+            månedsbeløp = vedtak.beregning.getMånedsberegninger().map {
+                Statistikk.Stønad.Månedsbeløp(
+                    måned = it.periode.fraOgMed.toString(),
+                    stonadsklassifisering = stønadsklassifisering(vedtak.behandling, it),
+                    bruttosats = it.getSatsbeløp().toLong(),
```

Comment:
Har dere avklart om de vil rundNearest eller roundDown?

Reply:
Skal vara roundDown. 😊 

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-double/to-long.html

Reply:
Greit det (selvom det høres litt rart ut), men da tenker jeg at vi bør kommentere at det er dette statistikk ønsker!